cds_dojson/marc21/fields/books/base.py

```diff
@@ -750,7 +750,7 @@ def alternative_titles(self, key, value):
 @out_strip
 def edition(self, key, value):
     """Translates edition indicator field."""
-    return clean_val('a', value, str)
+    return clean_val('a', value, str).replace("ed.", "")
```

Comment:
just in case you have `ed. 2` (with a space), WDYT?

```suggestion
    return clean_val('a', value, str).replace("ed.", "").strip()
```

Reply:
the decorator `@out_strip` does it automatically, so we don't need to remember to do it on each rule